testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -137,7 +142,8 @@ class AndroidSantaTrackerLintSmokeTest extends AndroidSantaTrackerSmokeTest {
         )
         SantaTrackerConfigurationCacheWorkaround.beforeBuild(checkoutDir, homeDir)
         runner.withArguments(runner.arguments + "--continue")
```

Comment:
:x: You're no longer updating the `result` variable, that's why CC asserts the build operations of the previous build.

```suggestion
        result = runner.withArguments(runner.arguments + "--continue")
```

Reply:
Thanks for the help with this!